From 0ce2caeb22ae391bf7f52ea88cd86d373a222859 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Steven!=20Ragnar=C3=B6k?= <steven@nuclearsandwich.com>
Date: Sat, 1 Jul 2017 01:30:31 -0400
Subject: [PATCH 2/2] Source environment in clean.

Even though it can fail during source builds we still want the
environment to run it during binarydeb jobs.
---
 debian/rules.em | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/rules.em b/debian/rules.em
index bab4eac..d953865 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -29,6 +29,7 @@ export @(var[0])=@(var[1])
 override_dh_auto_clean:
 	# We don't have ament_python in the iamge that creates sourcedebs
 	# So for now we need to give a pass to dh_auto_clean invoking setup.py clean.
+	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
 	dh_auto_clean || true
 
 override_dh_auto_configure:
-- 
2.7.4

